Controllers/DayOffRequestController.cs

```diff
@@ -0,0 +1,153 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.EntityFrameworkCore;
+using stunning_robot_HR.Data;
+using stunning_robot_HR.Models;
+
+namespace stunning_robot_HR.Controllers
+{
+    public class DayOffRequestController : Controller
+    {
+        private readonly stunning_robot_HRContext _context;
+
+        public DayOffRequestController(stunning_robot_HRContext context)
+        {
+            _context = context;
+        }
+
+        // GET: DayOffRequest
+        public async Task<IActionResult> Index()
+        {
+            return View(await _context.DayOffRequests.ToListAsync());
+        }
+
+        // GET: DayOffRequest/Details/5
+        public async Task<IActionResult> Details(int? id)
+        {
+            if (id == null)
+            {
+                return NotFound();
+            }
+
+            var dayOffRequest = await _context.DayOffRequests
+                .FirstOrDefaultAsync(m => m.RequestId == id);
+            if (dayOffRequest == null)
+            {
+                return NotFound();
+            }
+
+            return View(dayOffRequest);
+        }
+
+        // GET: DayOffRequest/Create
+        public IActionResult Create()
+        {
+            return View();
+        }
+
+        // POST: DayOffRequest/Create
+        // To protect from overposting attacks, enable the specific properties you want to bind to, for 
+        // more details, see http://go.microsoft.com/fwlink/?LinkId=317598.
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        public async Task<IActionResult> Create([Bind("RequestId,StartDayOfTimeRequest,EndDayOfTimeOffRequest,TotalNumberOfAvailableDaysOff,TotalNumberOfHoursWorked")] DayOffRequest dayOffRequest)
+        {
+            if (ModelState.IsValid)
+            {
+                _context.Add(dayOffRequest);
```

Comment:
You would often throw an add like this in a repository or even validation level class like fluid validation. Simply thinking there will be some validation down the road to ensure certain fields are correctly filled (validation class), that they have enough time to request (business class), that you persist the changes repository. It'll feel like overkill in the beginning, but as the project grows it'll allow you to ensure it stays maintainable.